example/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/1Password/connect-sdk-go/connect"
+	"github.com/1Password/connect-sdk-go/onepassword"
+	"os"
+	"time"
+)
+
+func main() {
```

Comment:
It would be great if this example also demonstrated the unmarshalling into a struct via annotations